rest-api/src/main/java/com/cloudbees/workflow/rest/external/ChangeSetExt.java

```diff
@@ -45,6 +45,14 @@
     private List<Commit> commits;
     private String consoleUrl; // Not a rest endpoint so not including in _links
 
+
+    /** Allows user to disable Jenkins user lookup for commit authors
+     * By setting System Property com.cloudbees.workflow.rest.external.ChangeSetExt.avoidResolvingCommitAuthors to 'true'
+     * This is a workaround for JENKINS-35484 where user lookup encounters issues */
+    public static boolean isResolvingCommitAuthors() {
```

Comment:
If it is a workaround (hopefully fixed in the right way later), better to keep this private so it can be removed at that point.